CONTRIBUTING.md

```diff
@@ -1,79 +1,17 @@
-
-## Conventional commits
-
-Our repositories follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification. Releasing to GitHub and NPM is done with the support of [semantic-release](https://semantic-release.gitbook.io/semantic-release/).
-
-Pull requests should have a title that follows the specification, otherwise, merging is blocked. If you are not familiar with the specification simply ask maintainers to modify. You can also use this cheatsheet if you want:
-
-- `fix: ` prefix in the title indicates that PR is a bug fix and PATCH release must be triggered.
-- `feat: ` prefix in the title indicates that PR is a feature and MINOR release must be triggered.
-- `docs: ` prefix in the title indicates that PR is only related to the documentation and there is no need to trigger release.
-- `chore: ` prefix in the title indicates that PR is only related to cleanup in the project and there is no need to trigger release.
-- `test: ` prefix in the title indicates that PR is only related to tests and there is no need to trigger release.
-- `refactor: ` prefix in the title indicates that PR is only related to refactoring and there is no need to trigger release.
-
-What about MAJOR release? just add `!` to the prefix, like `fix!: ` or `refactor!: `
-
-Prefix that follows specification is not enough though. Remember that the title must be clear and descriptive with usage of [imperative mood](https://chris.beams.io/posts/git-commit/#imperative).
-
-Happy contributing :heart:
-
-## License
-When you submit changes, your submissions are understood to be under the same [Apache 2.0 License](https://github.com/asyncapi/asyncapi/blob/master/LICENSE) that covers the project. Feel free to [contact the maintainers](https://www.asyncapi.com/slack-invite) if that's a concern.
-
-## References
-This document was adapted from the open-source contribution guidelines for [Facebook's Draft](https://github.com/facebook/draft-js/blob/master/CONTRIBUTING.md).
\ No newline at end of file
+- Make sure that extension is well documented with good examples,
+- Explain the use case,
+- Try to improve existing extensions, before you contribute a competing one,
+- Start discussion with an issue. Don't start with pull request first.
+- Every extension file needs to be located in [extensions folder](https://github.com/asyncapi/extensions-catalog/tree/master/extensions)
+- Observe the proper file naming convention, which calls for the lowercase name of the extension first.
+- Pull request with new or improved extension will be merged only if there is compliant implementation in the the [AsyncAPI JSON Schema](https://github.com/asyncapi/spec-json-schemas), [AsyncAPI JS Parser](https://www.github.com/asyncapi/parser-js) and the [AsyncAPI React component](https://github.com/asyncapi/asyncapi-react).
```

Comment:
```suggestion
- Pull request with new or improved extension will be merged only if there is compliant implementation in the the [AsyncAPI JSON Schema](https://github.com/asyncapi/spec-json-schemas), and the [AsyncAPI React component](https://github.com/asyncapi/asyncapi-react).
```

parser no longer required

also - please in this PR update https://github.com/asyncapi/extensions-catalog/blob/master/extensions/x.md so it also specifies version of the extension. Simple section and text like in this example https://github.com/asyncapi/bindings/tree/master/kafka

it is needed as you noticed when working on spec-json-schemas that we need to version schemas